apps/webshop/models.py

```diff
@@ -23,6 +23,22 @@ class Product(models.Model):
     deadline = models.DateTimeField(null=True, blank=True)
     active = models.BooleanField(default=True)
 
+    def calculate_stock(self):
+        if self.product_sizes.count() > 0:
+            return sum((product_size.stock for product_size in self.product_sizes.all()))
```

Comment:
Redundant parentheses? Generator expressions inside a method argument does not need wrapping.